TIKEBook_Lesson1.ipynb

```diff
@@ -0,0 +1,1247 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    plt.ylim(260000,266500) #zoom in on curve
</pre><p> </p><p>We should make this zoom the same as in the above plot</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>done</p>